charmcraft/templates/init/metadata.yaml.j2

```diff
@@ -1,10 +1,25 @@
 # Copyright {{ year }} {{ author }}
 # See LICENSE file for licensing details.
 name: {{ name }}
+display-name: |
+  TEMPLATE-TODO: fill out a display name for the Charmcraft store
 description: |
   TEMPLATE-TODO: fill out the charm's description
 summary: |
   TEMPLATE-TODO: fill out the charm's summary
+tags: |
+  TEMPLATE-TODO: fill out the tags your charm will present
+
+# TEMPLATE-TODO: replace with any services your charm provides to other charms and the interfaces it presents them on
+provides:
```

Comment:
This seems silly unless the underlying charm does something with it - I think we'd be better off with a comment at the top of the metadata.yaml template that points at the latest spec in the docs

I drafted a spec in a nice template format here: https://discourse.charmhub.io/t/charm-metadata-v2/3674/15?u=jnsgruk

Reply:
This is really great. We'll link to this (on discourse) until it makes it into the official docs

Reply:
It was my intention to make a page for it on the docs following review, I just hadn't yet got around to it ;) I thought in the 'Reference' section of the doc nav